bin/main/application.yml

```diff
@@ -0,0 +1,28 @@
+eureka:
+  client:
+    enabled: false
+    registerWithEureka: false
+  fetchRegistry: false
+
+# Server configuration
+server:
+  context-path: /micro
+  port: ${SERVICE_PORT:8082}
+
+management:
+  port: ${MANAGEMENT_PORT:8092}
+
+# Spring properties
+spring:
+  application:
+    name: customer
+    cloudant:
+      protocol: http
+      username: admin
+      password: passw0rd
+      host: 127.0.0.1
+      port: 5985
+      database: customers
+
+jwt:
+  sharedSecret: ${HS256_KEY:}
```

Comment:
Appsody gives us `application.properties`. To align with the format, you can use the same `.properties` format instead of using the yaml file

Reply:
resolved. 

`Did you include the deploy instructions in readme ?`

yes